next.config.js

```diff
@@ -4,6 +4,9 @@
 const nextConfig = {
   reactStrictMode: false,
   output: "export",
+  images: {
+    unoptimized: true,
+  },
```

Comment:
why optimize is disabled?

Reply:
and we need `output:"export"` so that we can build the project for gitpages